bin/dry-run.rb

```diff
@@ -804,7 +800,6 @@ def security_fix?(dependency)
 StackProf.results("tmp/stackprof-#{Time.now.strftime('%Y-%m-%d-%H:%M')}.dump") if $options[:profile]
 
 puts "🌍 Total requests made: '#{$network_trace_count}'"
-puts "🎈 Package manager version log: #{$package_manager_version_log.join('\n')}" if $package_manager_version_log.any?
```

Comment:
Should we replace this with the information now provided by the fetcher?

Reply:
Ah yes, of course!